cmd/ctr/commands/events/events.go

```diff
@@ -67,7 +70,8 @@ var Command = cli.Command{
 					e.Topic,
 					string(out),
 				); err != nil {
-					return err
+					log.G(ctx).WithError(err).Warn("cannot print event")
```

Comment:
above two "continue" LGTM, but this one. I prefer to keep it, "exit on any write error encountered", because it could be not temporary error. Should exit and check the error instead of ignoring it.

Reply:
reverted this line